yarn.lock

```diff
@@ -3443,6 +3483,11 @@ core-js-pure@^3.20.2:
   resolved "https://registry.yarnpkg.com/core-js-pure/-/core-js-pure-3.21.1.tgz#8c4d1e78839f5f46208de7230cebfb72bc3bdb51"
   integrity sha512-12VZfFIu+wyVbBebyHmRTuEE/tZrB4tJToWcwAMcsp3h4+sHR+fMJWbKpYiCRWlhFBq+KNyO8rIV9rTkeVmznQ==
 
+core-js-pure@^3.25.1, core-js-pure@^3.25.3:
```

Comment:
This second version of core-js-pure is also resolved by upgrading @testing-library/react

Reply:
This out of scope for now